content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -120,95 +134,51 @@ using the (deprecated) v1alpha2 API instead.
 
 ### containerd
 
-This section contains the necessary steps to use containerd as CRI runtime.
+This section outlines the necessary steps to use containerd as CRI runtime.
```

Comment:
should it be `containerd`?

Reply:
We don't need code style for the tool name (which is containerd).